tests/restart/incompressible_navier_stokes.mpirun=2.output

```diff
@@ -0,0 +1,2684 @@
+
+  Max. number of refinements:                1
+  Number of cells:                           4
+
+Construct incompressible Navier-Stokes operator ...
+Velocity:
+  degree of 1D polynomials (normal):         15
+  degree of 1D polynomials (tangential):     14
+  number of dofs per cell:                   480
+  number of dofs (total):                    1860
+Pressure:
+  degree of 1D polynomials:                  14
+  number of dofs per cell:                   225
+  number of dofs (total):                    900
+Velocity and pressure:
+  number of dofs per cell:                   705
+  number of dofs (total):                    2760
+
+... done!
+
+Setup incompressible Navier-Stokes operator ...
+
+... done!
+
+Setup Multistep time integrator ...
+
+________________________________________________________________________________
+
+ Reading restart file:
+    vector global l2 norm: 3.11102677e+01      
+    vector global l2 norm: 3.11103877e+01      
+    vector global l2 norm: 6.33036869e-01      
+    vector global l2 norm: 6.33041753e-01      
+    vector global l2 norm: 0.00000000e+00      
+    vector global l2 norm: 0.00000000e+00      
+    vector global l2 norm: 3.06895467e+01      
+    vector global l2 norm: 3.06896651e+01      
+
+ ... done!
+________________________________________________________________________________
+
+
+... done!
+
+Starting time loop ...
+
+Calculate error for velocity at time t = 1.2500e-02:
+  Relative error (L2-norm): 5.33532e-08
+
+Calculate error for pressure at time t = 1.2500e-02:
+  Relative error (L2-norm): 5.83819e-07
+
+________________________________________________________________________________
+
+ Time step number = 1       t = 1.25000e-02 -> t + dt = 1.28125e-02
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 9       t = 1.50000e-02 -> t + dt = 1.53125e-02
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 17      t = 1.75000e-02 -> t + dt = 1.78125e-02
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 25      t = 2.00000e-02 -> t + dt = 2.03125e-02
+________________________________________________________________________________
+
+________________________________________________________________________________
+
+ Time step number = 33      t = 2.25000e-02 -> t + dt = 2.28125e-02
+________________________________________________________________________________
+
+Calculate error for velocity at time t = 2.5000e-02:
+  Relative error (L2-norm): 5.52249e-05
+
+Calculate error for pressure at time t = 2.5000e-02:
+  Relative error (L2-norm): 1.56751e-05
```

Comment:
and get 5e-5 and 2e-5

so a bit is for sure the projection, BUT below we start with `HDIV`